tests/network/l2_bridge/utils.py

```diff
@@ -208,20 +208,21 @@ def hot_plug_interface_and_set_address(
     ipv4_address,
     sriov=False,
 ):
-    hot_plug_interface(
+    iface = hot_plug_interface(
         vm=vm,
         hot_plugged_interface_name=hot_plugged_interface_name,
         net_attach_def_name=net_attach_def_name,
         sriov=sriov,
     )
-    wait_for_interface_hot_plug_completion(vmi=vm.vmi, interface_name=hot_plugged_interface_name)
```

Comment:
Why has this been removed?

Reply:
Makes sense, thanks.

Then the existence of the other usages of this helper hints they also need cleanup treatment in a follow up.